client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,84 @@
+<div class="widget-presentation">
+
+    <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title"></h2>
+
+    <tuc-toast-message></tuc-toast-message>
+
+    <div data-ng-if="$ctrl.isRMAOpened" class="mb-3">
+        <span class="row" data-translate="xdsl_access_comfort_exchange_already_opened_rma"></span>
+        <div data-ng-repeat="rma in $ctrl.rmas track by $index">
```

Comment:
```suggestion
        <div data-ng-repeat="rma in ::$ctrl.rmas track by $index">
```
if `$ctrl.rmas` will not change :)